backend/src/packages/users/user-details.model.ts

```diff
@@ -0,0 +1,32 @@
+import { Model } from 'objection';
+
+import {
+  AbstractModel,
+  DatabaseTableName,
+} from '#libs/packages/database/database.js';
+
+import { type RelationMappingsType } from './libs/types/types.js';
+import { UserModel } from './user.model.js';
+
+class UserDetailsModel extends AbstractModel {
+  public fullName!: string;
+
+  public static override get tableName(): string {
+    return DatabaseTableName.USER_DETAILS;
+  }
+
+  public static get relationMappings(): RelationMappingsType {
+    return {
+      user: {
+        relation: Model.BelongsToOneRelation,
+        modelClass: UserModel,
+        join: {
+          from: 'user_details.user_id',
```

Comment:
Use enums here

Reply:
Do I need to use enums both for table name and column name? In mini project only table name is used, but I think It's ok to create enum for column names for eact table.

Reply:
Yep, for columns too)